### PR TITLE
readme: tool lints are stable now

### PR DESCRIPTION
**Pull request by phansch**

### CODE REVIEW

**Comment by matthiaskrgr** [comment]
Nit: perhaps throw in a quick note that this also works with lint groups? (like `clippy::pedantic`)
LGTM besides that. :)

File: README.md [hunk 1]
```diff
@@ -149,21 +149,7 @@ You can add options to your code to `allow`/`warn`/`deny` Clippy lints:
...
-
-For this to work you have to use Clippy on the nightly toolchain: `cargo +nightly clippy`. If you
-want to use Clippy with the stable toolchain, you can stick to the old unscoped method to
-enable/disable Clippy lints until `tool_lints` are stable:
-```rust
-#![cfg_attr(feature = "cargo-clippy", allow(clippy_lint))]
-```
-
-If you do not want to include your lint levels in your code, you can globally enable/disable lints by passing extra flags to clippy during the run: `cargo clippy -- -A lint_name` will run clippy with `lint_name` disabled and `cargo clippy -- -W lint_name` will run it with that enabled. On newer compilers you may need to use `clippy::lint_name` instead.
+If you do not want to include your lint levels in your code, you can globally enable/disable lints by passing extra flags to clippy during the run: `cargo clippy -- -A clippy::lint_name` will run clippy with `lint_name` disabled and `cargo clippy -- -W clippy::lint_name` will run it with that enabled. This also works with lint groups. For example you can run Clippy with warnings for all lints enabled: `cargo clippy -- -W clippy::pedantic`
```
[the remark this answers is not in the document]
Comment (phansch) [reply README.md]: You're right =) I also changed it in some more places in 0c93e4c just now

[the remark this answers is not in the document]
Comment (phansch) [reply README.md]: Now I see it still mentions `nightly` there.. I will fix that too while I'm at it.

File: README.md [hunk 2]
```diff
@@ -106,18 +106,6 @@ script:
...
 
-It might happen that Clippy is not available for a certain nightly release.
-In this case you can try to conditionally install Clippy from the git repo.
-
-```yaml
-language: rust
-rust:
-  - nightly
-before_script:
-   - rustup component add clippy --toolchain=nightly || cargo install --git https://github.com/rust-lang/rust-clippy/ --force clippy
```
Comment (matthiaskrgr) [inline README.md]: Hmm, I think this is still useful for people who want to run clippy on the nightly channel instead of beta or stable.

Comment (matthiaskrgr) [reply README.md]: As in, this hack *was* useful before it was broken (#3523) for people who didn't want to wait for new rust releases to get clippy updates.

Comment (phansch) [reply README.md]: removed that commit again, should be fine now :+1:

Comment (aldanor) [reply README.md]: @phansch Is #3523 still meant to fail? Because it does.

**Pull request merged**